libs/labelbox/src/labelbox/schema/member.py

```diff
@@ -0,0 +1,433 @@
+from typing import Optional
+from labelbox.exceptions import (
+    MalformedQueryException,
+    ResourceNotFoundError,
+    UnprocessableEntityError,
+)
+from typing import Set, Iterator, Any, List
+from pydantic import (
+    Field,
+    field_validator,
+    model_serializer,
+    model_validator,
+)
+from labelbox.utils import _CamelCaseMixin
+from labelbox.schema.role import Role
+from labelbox import Client
+
+
+class ProjectMembership(_CamelCaseMixin):
```

Comment:
https://github.com/Labelbox/labelbox-python/blob/6fb945c73baf94b428de3a234e6fe9a315ebd7a5/libs/labelbox/src/labelbox/schema/project.py#L1652
This won't work for this class since the relationships are off. Plus, ProjectRole and ProjectMemberships are technically two different objects in our backend.